README.md

```diff
@@ -74,6 +74,36 @@ Then, whenever we want to publish a metric, we simply do:
 myMetric.put(value, metric, additionalDimensions);
 ```
 
+### Using summary metrics
+
+Instead of sending individual data points for your metric, you may want to send
+summary metrics. Summary metrics track statistics over time, and send those
+statistics to CloudWatch on a configurable interval. For instance, you might
+want to know your total network throughput, but you don't care about individual
+request size percentiles. You could use `summaryPut` to track this data and send
+it to CloudWatch with fewer requests:
+
+```js
+var metric = new cloudwatchMetrics.Metric('namespace', 'Bytes');
+
+function onRequest(req) {
+	// This will still track maximum, minimum, sum, count, and average, but won't
+	// take up lots of CloudWatch requests doing so.
+	metric.summaryPut(req.size, 'requestSize');
+}
+```
+
+Note that metrics use different summaries for different dimensions, _and that
+the order of the dimensions is significant!_ In other words, these track
```

Comment:
Is there a use case for this? Wouldn't it be easy to order the dimensions e.g. alphabetically when generating the key?

Reply:
it probably would, but this makes implementation simpler, and it's IMO reasonable to expect the consumer of this library to always pass dimensions in the same order.